app/models/host.rb

```diff
@@ -181,20 +181,31 @@ class Host < ApplicationRecord
 
   supports     :destroy
   supports_not :quick_stats
-  supports     :reset do
-    unsupported_reason_add(:reset, _("The Host is not configured for IPMI")) if ipmi_address.blank?
-    unsupported_reason_add(:reset, _("The Host has no IPMI credentials")) if authentication_type(:ipmi).nil?
-    if authentication_userid(:ipmi).blank? || authentication_password(:ipmi).blank?
-      unsupported_reason_add(:reset, _("The Host has invalid IPMI credentials"))
-    end
-  end
   supports_not :refresh_advanced_settings
   supports_not :refresh_firewall_rules
   supports_not :refresh_logs
   supports_not :refresh_network_interfaces
   supports_not :set_node_maintenance
   supports_not :smartstate_analysis
   supports_not :unset_node_maintenance
+  supports_not :reboot
+  supports_not :shutdown
+  supports_not :standby
+  supports_not :enter_maint_mode
+  supports_not :exit_maint_mode
+  supports_not :enable_vmotion
+  supports_not :disable_vmotion
+  supports_not :vmotion
+
+  supports_not :start
+  supports_not :stop
+  supports     :reset do
```

Comment:
TANGENT: wow. reset seems expensive

TANGENT2: also seems to duplicate `validate_ipmi`